src/decorators.ts

```diff
@@ -36,8 +42,39 @@ export const Command =
         _propertyKey: string,
         descriptor: TypedPropertyDescriptor<CommandHandler>
     ) => {
-        if (descriptor.value)
-            registry.commands.set(options.name, descriptor.value);
+        const handler = descriptor.value;
+        if (handler) {
+            const wrapped = async (interaction: CommandInteraction) => {
+                try {
+                    await handler(interaction);
+                } catch (e) {
+                    if (e instanceof BotError) {
+                        const embed = new MessageEmbed()
+                            .setColor("#ec3030")
+                            .setDescription(e.message);
+                        await interaction.reply({
+                            embeds: [embed],
+                            ephemeral: true,
```

Comment:
i dont know if they should all necessarily be emphemeral -- like for the ones in the bot command channel. if theres a real issue like the FE api being down it'd seem silly for no one to know and everyone has to try to find out.